src/ui/control/attribution_control.js

```diff
@@ -186,6 +174,14 @@ class AttributionControl {
             return true;
         });
 
+        if (this.options.customAttribution) {
+            if (Array.isArray(this.options.customAttribution)) {
+                attributions = [...this.options.customAttribution.filter(s => typeof s === 'string'), ...attributions];
```

Comment:
BTW, I'm not sure why we did this filter in the first place — it might no longer be relevant, possibly worth removing. Originally introduced in #7444

Reply:
without the check for string type, any data type can be passed through customAttribution, but it still appears fine displayed in the attribution if we want to allow for that?

Reply:
In theory it shouldn't matter, because we later do `attributions.join(' | ')` which automatically coerces each item into string.

Reply:
Agreed, I removed the string filtering, just pushed the commit